small-talk/pages/api/auth2.js

```diff
@@ -22,9 +22,21 @@ export default async function POST(req,res)
         session_id : session_id
     }
     const collectionName = "patients"
-    await mongoboi.connect()
-    const result = await mongoboi.findOne("patients", query)
-    await mongoboi.disconnect()
-    if (result == null) { res.status(200).json({ found: false }); return; }
-    res.status(200).json({ found: true })
-}//
\ No newline at end of file
+    try 
+    {
+        await mongoboi.connect()
+        const result = await mongoboi.findOne("patients", query)
+    } catch (error) 
```

Comment:
I like that we're implementing error catching here. Makes it better for us to see what is going on instead of having no feedback from the application